drivers/flash/spi_flash_sfdp.c

```diff
@@ -1,558 +1,993 @@
 /*
  * Copyright (c) 2017 Crypta Labs Ltd
+ * Copyright (c) 2018 Findlay Feng
```

Comment:
Merge this specific modification of sfdp to your first commit.

And this actual commit becomes one for adding microchip table support then.

Reply:
I will change it